ui/imports/shared/stores/send/TransactionStore.qml

```diff
@@ -324,12 +324,15 @@ QtObject {
             },
             FastExpressionFilter {
                 expression: {
+                    if (!root.walletAssetStore.assetsController.filterAcceptsSymbol(model.symbol)) // explicitely hidden
```

Comment:
Don't we need to add a dependency on the ManageTokensController::revision so that the `filterAcceptsSymbol` to be reassessed when the settings change?

Reply:
Not necessarily, since this is happening inside the popup, which is reopen (and thus reevaluated) every time

Reply:
But can't hurt, going to add it :)

Reply:
Done

Reply:
> But can't hurt, going to add it :)

Yeah, I think so. Eventually we'll have the synced settings, so it should cover this.